examples/nft/scripts/transfer/create-transfer-nft.js

```diff
@@ -0,0 +1,55 @@
+const { executeTransaction } = require("./common");
+const { TransactionType, SignType } = require("algob");
+
+async function run(runtimeEnv, deployer) {
+
+  const masterAccount = deployer.accountsByName.get("master-account")
+  const johnAccount = deployer.accountsByName.get("john-account");
+
+  const sscInfo = await deployer.getSSC("nft_approval.py", "nft_clear_state.py");
+  const appId = sscInfo.appID;
+  console.log(sscInfo);
+
+  const nft = { name: "some-nft", ref: "ref" }
+  //push arguments "create" and nft data 
+  let appArgs = [
+    new Uint8Array(Buffer.from("create")),
+    new Uint8Array(Buffer.from(nft.name)),
+    new Uint8Array(Buffer.from(nft.ref))
```

Comment:
Let's create a function for this, and use it everywhere (you can put it somewhere to `alob/src/lib`):
```
function toBytes(s: string): Uint8Array {
    return new Uint8Array(Buffer.from(s))
}
```

Reply:
ok